articles/cognitive-services/Translator/tutorial-wpf-translation-csharp.md

```diff
@@ -419,7 +420,7 @@ private string CorrectSpelling(string text)
 
     // Create a request to Bing Spell Check API
     HttpWebRequest spellCheckWebRequest = (HttpWebRequest)WebRequest.Create(uri);
-    spellCheckWebRequest.Headers.Add("Ocp-Apim-Subscription-Key", COGNITIVE_SERVICES_KEY);

```

Comment:
Cognitive services key should be used like this:

spellCheckWebRequest.Headers.Add("Ocp-Apim-Subscription-Key", COGNITIVE_SERVICES_KEY);
 spellCheckWebRequest.Headers.Add("Ocp-Apim-Subscription-Region", "REGION");

Can you please revert?

Reply:
@swmachan - I thought only Translator Text required the user to explicitly provide the region header. 

@KSchlobohm - I just tested this [code](https://github.com/MicrosoftTranslator/Text-Translation-API-V3-C-Sharp-Tutorial) using the Cognitive Services key and was able to run all tasks including spell check/correction. I did however, find a spacing issue in the code sample in the tutorial that was causing it to break and 400. I'm in the process of updating the doc now, but can you confirm that the code I've shared works for you? Thanks!